scania-digital-design-system/tegel
# fix(core): removing concurrently and tegel script

**Describe pull-request**  
Removing concurrently from the core package

**Solving issue**  
Seems like concurrently is not needed anymore on the core level package as root level one has it already and seems to work OK. Starting system now happens from the root level so it seems that `tegel` script in core package is not needed any more ( + it trows an error too )

**How to test**  
1. Drop node modules from core
2. run `npm i` on core level
3. run `npm run build` on core level
4. run `npm run storybook` on root level
5. Storybook and Stencil should work as expected


**Additional context**  
_Add any other context about the pull-request here._
